src/allotropy/allotrope/models/adm/liquid_chromatography/rec/_2023/_09/liquid_chromatography.py

```diff
@@ -227,12 +208,31 @@ class PeakItem:
     asymmetry_factor_measured_at_4_4___height: TQuantityValueUnitless | None = None
     asymmetry_factor_measured_at_baseline: TQuantityValueUnitless | None = None
     chromatographic_peak_asymmetry_factor: TQuantityValueUnitless | None = None
-    field_index: int | None = None
+    chromatographic_peak_resolution_using_peak_width_at_half_height: TQuantityValueUnitless | None = (
+        None
+    )
+    chromatographic_peak_resolution_using_statistical_moments: TQuantityValueUnitless | None = (
+        None
+    )
+    number_of_theoretical_plates__chromatography_: TQuantityValueUnitless | None = None
+    number_of_theoretical_plates_measured_at_60_7___of_peak_height: TQuantityValueUnitless | None = (
+        None
+    )
+    number_of_theoretical_plates_measured_at_32_4___of_peak_height: TQuantityValueUnitless | None = (
+        None
+    )
+    number_of_theoretical_plates_measured_at_13_4___of_peak_height: TQuantityValueUnitless | None = (
+        None
+    )
+    number_of_theoretical_plates_measured_at_4_4___of_peak_height: TQuantityValueUnitless | None = (
+        None
+    )
+    number_of_theoretical_plates_by_tangent_method: TQuantityValueUnitless | None = None
 
 
 @dataclass(kw_only=True)
 class PeakList:
-    peak: list[PeakItem] | None = None
+    peak: list[PeakItem] | list[Peak] | None = None
```

Comment:
NOTE to reviewer: this is a consequence of removing anyof combination, and it's completely desirable, however it is still correct.

The ideal here would be to combine `Peak` and `PeakItem`, but right now we only combine if the names are of the form `BaseClass[0-9]*`

I need to experiment with including `Item` as an option, and will do in a separate PR.

For now, this schema is not used yet, so I'd rather prioritize making the new `plate-reader` work than worry about this.